server/lib/fusor/multilog.rb

```diff
@@ -35,6 +36,33 @@ def detach_all
     end
   end
 
+  def collect(base_path)
+    if check_nested_key(SETTINGS, [:fusor, :system, :logging, :watch])
+      SETTINGS[:fusor][:system][:logging][:watch].each do |entry|
+        file = entry[:file]
+
+        if !@watchlist.key? file
+          # make containing folder
+          *x, _ = file.split("/")
+          path = File.join(base_path, x.join("/"))
```

Comment:
Any particular reason you're doing `file.split` and then `x.join("/")`? Seems like `File.dirname(file)` would be simpler.